demos/projects/ST/b-l475e-iot01a/config/demo_config.h

```diff
@@ -506,4 +506,10 @@ static unsigned char root_cert_array[] = {
  */
 #define WIFI_SECURITY_TYPE  WIFI_ECN_WPA2_PSK
 
+#define democonfigADU_DEVICE_MANUFACTURER "ST"
+#define democonfigADU_DEVICE_MODEL        "STM32L496"
```

Comment:
Should these be `L475`?

Reply:
wow, clearly cellular is still in my muscle memory haha